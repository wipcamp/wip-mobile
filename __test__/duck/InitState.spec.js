/**
 * @jest-environment node
 */

import reducer, {
    initState,
    SET_INIT
} from '../../ducks/InitState'

describe('action', () => {
    it('SET_INIT', () => {
        expect(SET_INIT()).toMatchSnapshot()
    })
})

describe('reducer', () => {
    it('SET_INIT', () => {
        expect(reducer(initState, SET_INIT())).toMatchSnapshot()
    })

    it('default', () => {
        expect(reducer(initState, {
            type: 'default'
        })).toMatchSnapshot()
    })
})