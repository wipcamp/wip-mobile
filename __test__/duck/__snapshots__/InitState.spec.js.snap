// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action SET_INIT 1`] = `
Object {
  "type": "INIT_STATE",
}
`;

exports[`reducer SET_INIT 1`] = `
Object {
  "categoryProblem": Array [],
  "newProblem": Object {
    "description": "",
    "priority_id": 0,
    "problem_type_id": 0,
    "report_id": null,
    "topic": "",
  },
  "notification": Array [],
  "priority": Array [],
  "problem": Array [],
  "timetable": Array [],
}
`;

exports[`reducer default 1`] = `
Object {
  "categoryProblem": Array [],
  "newProblem": Object {
    "description": "",
    "priority_id": 0,
    "problem_type_id": 0,
    "report_id": null,
    "topic": "",
  },
  "notification": Array [],
  "priority": Array [],
  "problem": Array [],
  "timetable": Array [],
}
`;
